shared/test/test_credentials.json.sample

```diff
@@ -0,0 +1,27 @@
+{
+
+"access_token":"__NOT_REQUIRED__",
+ 
+"test_client_id":"__INSERT_REMOTE_ACCESS_CLIENT_KEY_HERE__",
+
+"test_login_domain":"test.salesforce.com",
+
+"test_redirect_uri":"__INSERT_REMOTE_ACCESS_CALLBACK_URL_HERE__",
+
+"refresh_token":"__INSERT_TOKEN_HERE__",
+ 
+"instance_url":"__INSERT_INSTANCE_URL_HERE__",
+
+"identity_url":"__INSERT_IDENTITY_URL_HERE__"
+
+"organization_id": "__INSERT_ORG_ID_HERE__",
```

Comment:
Note the extra fields.

To create a test_credentials.json use the [credsHelper](https://github.com/forcedotcom/SalesforceMobileSDK-Shared/tree/dev/credshelper)